tests/unit_tests/repobee/test_main.py

```diff
@@ -363,7 +363,7 @@ def test_does_not_log_error_when_command_is_used_incorrectly(mocker):
 
     with pytest.raises(SystemExit):
         # note that the category is misspelled
-        main.main("repbee isues -h".split())
+        main.main("repbee issues -h".split())
```

Comment:
This is intentionally misspelled as part of a regression test :)

```suggestion
        main.main("repbee isues -h".split())
```